clients/admin-ui/src/features/data-catalog/projects/CatalogProjectsTable.tsx

```diff
@@ -0,0 +1,190 @@
+    maxW="70%"
+    data-testid="empty-state"
+    alignSelf="center"
+    margin="auto"
+  >
+    <VStack>
+      <Text fontSize="md" fontWeight="600">
+        No resources found
+      </Text>
+    </VStack>
+  </VStack>
+);
+
+const columnHelper = createColumnHelper<StagedResourceAPIResponse>();
+
+const CatalogProjectsTable = ({
+  systemKey,
+  monitorConfigIds,
+}: {
+  systemKey: string;
+  monitorConfigIds: string[];
+}) => {
+  const [rowSelectionState, setRowSelectionState] = useState<RowSelectionState>(
+    {},
+  );
+
+  const {
+    PAGE_SIZES,
+    pageSize,
+    setPageSize,
+    onPreviousPageClick,
+    isPreviousPageDisabled,
+    onNextPageClick,
+    isNextPageDisabled,
+    startRange,
+    endRange,
+    pageIndex,
+    setTotalPages,
+  } = useServerSidePagination();
+
+  const {
+    isFetching,
+    isLoading,
+    data: queryResult,
+  } = useGetCatalogProjectsQuery({
+    page: pageIndex,
+    size: pageSize,
+    monitor_config_ids: monitorConfigIds,
+  });
+
+  const router = useRouter();
+
+  const {
+    items: data,
+    total: totalRows,
+    pages: totalPages,
+  } = useMemo(() => queryResult ?? EMPTY_RESPONSE, [queryResult]);
+
+  useEffect(() => {
+    setTotalPages(totalPages);
+  }, [totalPages, setTotalPages]);
+
+  const columns: ColumnDef<StagedResourceAPIResponse, any>[] = useMemo(
+    () => [
+      columnHelper.accessor((row) => row.name, {
+        id: "name",
+        cell: (props) => (
+          <CatalogResourceNameCell resource={props.row.original} />
+        ),
+        header: "Project",
+      }),
+      columnHelper.display({
+        id: "status",
+        cell: ({ row }) => (
+          <CatalogStatusBadgeCell
+            status={getCatalogResourceStatus(row.original)}
+          />
+        ),
+        header: "Status",
+      }),
+      columnHelper.accessor((row) => row.monitor_config_id, {
+        id: "monitorConfigId",
+        cell: (props) => <DefaultCell value={props.getValue()} />,
+        header: "Detected by",
+      }),
+      columnHelper.accessor((row) => row.description, {
+        id: "description",
+        cell: (props) => <DefaultCell value={props.getValue()} />,
+        header: "Description",
+      }),
+      columnHelper.accessor((row) => row.updated_at, {
+        id: "lastUpdated",
+        cell: (props) => <RelativeTimestampCell time={props.getValue()} />,
+        header: "Updated",
+        meta: {
+          cellProps: {
+            borderRight: "none",
+          },
+        },
+      }),
+    ],
+    [],
+  );
+
+  const tableInstance = useReactTable<StagedResourceAPIResponse>({
+    getCoreRowModel: getCoreRowModel(),
+    getGroupedRowModel: getGroupedRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    manualPagination: true,
+    columnResizeMode: "onChange",
+    columns,
+    data,
+    onRowSelectionChange: setRowSelectionState,
+    state: {
+      rowSelection: rowSelectionState,
+    },
+  });
+
+  if (isLoading || isFetching) {
```

Comment:
I would suggest only using `isLoading` here so the skeleton appears on initial load, but not every time a new Data Use is added.